public/modules/interface/navbar/script.tsx

```
@@ -1,61 +1,94 @@
+//  NAVBAR
+//
+
+// NAVBAR -> DISPLAY STATUS
+let navbarState = true;
+
+// NAVBAR -> OPENER FUNCTION
+function alternate(event) {
+    if (navbarState) {
+        var navbarStateChange = !(event.clientX <= window.innerWidth * 0.06)
+    } else {
+        var navbarStateChange = event.clientX <= window.innerWidth * 0.01;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'navbarStateChange'. Consider to delete it or rename it.

'navbarStateChange' is defined here:


(lint/suspicious/noRedeclare)